### PR TITLE
Feature/search volunteers fixes

### PR DESCRIPTION
**Pull request by ofekudi**
Screenshot 2024-10-05 at 16 31 32
Screenshot 2024-10-05 at 16 30 06

### CODE REVIEW

File: src/app/asign-family/asign-family.component.scss [hunk 1]
```diff
@@ -103,4 +103,21 @@ h5 {
 
 ::ng-deep .mat-form-field-infix {
   width: auto !important;
-}
\ No newline at end of file
+}
+
+.search-autocomplete-option {
+  position: absolute;
```
Comment (ofekudi) [inline src/app/asign-family/asign-family.component.scss:109]: I couldn't find a way of extending the span with classname of "mdc-list-item__primary-text" so I had to do this ugly absolute thing :( 
Lmk if you have any other way

File: src/app/asign-family/asign-family.component.ts [hunk 2]
```diff
@@ -124,6 +124,15 @@ export class AsignFamilyComponent implements OnInit, OnDestroy {
     if (this.identifier != this.familyLists.helper.phone.thePhone)
       this.identifier = this.familyLists.helper.phone.thePhone
   }
+
+  getKeyboardType() {
+    return sessionStorage.getItem('keyboardType') || 'text'
+  }
+
+  toggleKeyboardType() {
+    sessionStorage.setItem('keyboardType', this.getKeyboardType() === 'text' ? 'tel' : 'text')
```
Comment (ofekudi) [inline src/app/asign-family/asign-family.component.ts:133]: Saving type for the entire session.

File: src/app/asign-family/asign-family.component.ts [hunk 3]
```diff
@@ -124,6 +124,15 @@ export class AsignFamilyComponent implements OnInit, OnDestroy {
     if (this.identifier != this.familyLists.helper.phone.thePhone)
       this.identifier = this.familyLists.helper.phone.thePhone
   }
+
+  getKeyboardType() {
+    return sessionStorage.getItem('keyboardType') || 'text'
+  }
+
+  toggleKeyboardType() {
```
Comment (ofekudi) [inline src/app/asign-family/asign-family.component.ts:132]: It works for me without adding more logic for blur/focus of the keyboard, so lmk if it also works for you or should I add logic for it

Comment (noam-honig) [reply src/app/asign-family/asign-family.component.ts:132]: I tested it on my iphone - and the keyboard doesn't change (just the alignment changes) - can you try the blur / focus?

**Pull request merged**